CHANGELOG.md

```diff
@@ -335,11 +337,11 @@ The binaries published with this release are built with Go1.17.8 to avoid [CVE-2
 
 ### Changed
 
--
+- 
```

Comment:
Can you try removing this white-space. CI must be failing for because of these white-space